src/main/java/uk/gov/hmcts/reform/cpo/validators/ValidationError.java

```diff
@@ -14,6 +14,11 @@
     public static final String ORDER_REFERENCE_REQUIRED = "Order Reference Party is required.";
     public static final String RESPONSIBLE_PARTY_REQUIRED = "Responsible Party is required.";
 
+    public static final String IDS_EMPTY = "IDs are required";
+    public static final String CASE_ID_INVALID_LENGTH = "Case ID has to be 16-digits long";
```

Comment:
This should not be needed after move to single Luhn and length check in `@ValidCaseId`

Reply:
Resolving as code no longer applicable after recent refactored of `DELETE` to mirror `GET` endpoint validation functionality